.github/workflows/build_bazel.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 The StableHLO Authors.
+
+name: Bazel Build
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+  schedule:
+    # Run build_bazel workflow once a day
+    - cron:  '0 12 * * *'
+  workflow_dispatch:
+
+# Ensure that only a single job or workflow using the same
+# concurrency group will run at a time. This would cancel
+# any in-progress jobs in the same github workflow and github
+# ref (e.g. refs/heads/main or refs/pull/<pr_number>/merge).
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  bazel-build:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: Checkout StableHLO
+      uses: actions/checkout@v2
+
+    - name: Running lint_llvm_commit.sh on source files
```

Comment:
Other names in this file use imperative mood. How about "Run lint_llvm_commit.sh"?

Reply:
Agree, also "on source files" seems incorrect. Could also title it "Validate LLVM Commit" if we want to avoid mentioning the bash script by name.